client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -111,6 +114,35 @@ export const Sync: React.FC = () => {
         </Row>
       </Grid>
       <SyncProgress syncStatus={syncStatus} isOperational={true} />
+      <Grid
+        container
+        flexDirection="column"
+        justifyContent="flex-start"
+        style={{ padding: '15 15 50 15', minWidth: 650 }}
+        marginTop={2}
+        flexWrap="nowrap"
+      >
+        <Typography variant="h5" color="primary" style={{ paddingBottom: 25 }}>
+          {t('heading.user-sync')}
+        </Typography>
+        <Row title={t('sync-info.last-successful-sync')}>
+          <FormattedSyncDate
+            date={DateUtils.getDateOrNull(data?.lastSuccessfulSync || null)}
+          />
+        </Row>
+        <Row>
+          <LoadingButton
+            isLoading={updateUserIsLoading}
+            startIcon={<RadioIcon />}
+            variant="contained"
+            sx={{ fontSize: '12px' }}
+            disabled={false}
+            onClick={async () => await updateUser()}
```

Comment:
An error please if server is not reachable, similar to what Sync Status shows, error wrapper with Unable to connect to server

Reply:
thanks - was thinking the same thing; got caught by this earlier - even though the sync showed an error and once I'd thought about it, it was pretty obvs that the central server was unreachable, I still pushed the button a number of times expecting something to happen.

Reply:
[8b6407f](https://github.com/openmsupply/open-msupply/pull/2080/commits/8b6407f31d5863927f824611bbb3a8e09a107cab)